src/lib/transition_frontier/persistent_root/persistent_root.ml

```diff
@@ -103,11 +103,11 @@ module Instance = struct
             Some root_identifier )
 
   let set_root_state_hash t state_hash ~genesis_state_hash =
-    let root_identifier =
+    let _root_identifier =
```

Comment:
We should delete this assignment, and we can then remove the `~genesis_state_hash` argument to this function as well.